src/components/CredentialCard/CredentialCard.tsx

```diff
@@ -258,7 +264,7 @@ function DownloadModal({
                 className={styles.toggle}
                 type="checkbox"
                 defaultChecked={false}
-                onClick={handleToggle}
+                onChange={handleToggle}
```

Comment:
I have a very old and vague memory that onChange on checkboxes worked differently in some browsers, not at the time of click but at the time of blur. Could you confirm that with this change the behavior in Firefox is the same as in Chrome?

Reply:
Looked into it, it was only in IE that checkboxes behaved differently with onChange